CODEOWNERS

```diff
@@ -368,6 +368,7 @@
 /subsys/testsuite/                        @nashif
 /subsys/usb/                              @jfischer-phytec-iot @finikorg
 /tests/                                   @nashif
+/tests/application_development/libcpp/    @pabigot
```

Comment:
libcpp may sounds weird to some, `cpp` is C preprocessor, and C++'s standard lib is called libstdc++ (but nope, don't call it like that, Windows and stuff).

Reply:
I've never liked using `cpp` for C++; I usually use `cxx` but `cpp` was existing practice.  I've renamed it to `libcxx`.

Reply:
> I've renamed it to libcxx.

That's definitely better, thanks.